packages/react-core/src/components/Chip/ChipGroup.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import styles from '@patternfly/react-styles/css/components/ChipGroup/chip-group';
+import styles from '@patternfly/react-styles/css/components/Chip/chip-group';
```

Comment:
Had to update this because of this change: https://github.com/patternfly/patternfly/pull/5559